packages/build/tests/error/snapshots/tests.js.md

```diff
@@ -2302,6 +2302,99 @@ Generated by [AVA](https://avajs.dev).
       }␊
     }`
 
+## Report API error
```

Comment:
This is an unrelated test, which was not present in the test snapshots, for some reason :thinking: 
Running `ava -u` fixed this.